stories/Welcome.mdx

```diff
@@ -1,7 +1,158 @@
-import './styles.css';
+import './welcome-styles.css';
+import gridironLogo from './assets/gridironSurvivorLogo.svg';
+import appwriteLogo from './assets/appwriteLogo.svg';
+import blogRecorderLogo from './assets/blogRecorderLogo.svg';
+import frontendMentorLogo from './assets/frontendMentorLogo.svg';
+import gitkrakenLogo from './assets/gitkrakenLogo.svg';
+import pastelLogo from './assets/pastelLogo.svg';
+import Image from 'next/image';
+import { ArrowRight } from 'lucide-react';
+import { Github } from 'lucide-react';
 
-<h1>Welcome</h1>
+import { Button } from '../components/Button/Button.tsx';
 
-<div className="doc_nav_links">
-  <p>[Application Setup](?path=/docs/about-application-setup--docs)</p>
-</div>
+<html>
+  <body class="dark bg-background text-muted-foreground max-w-screen-lg mx-auto text-pretty flex flex-col space-y-36 pb-36">
```

Comment:
<img width="960" alt="Screenshot 2024-10-01 140118" src="https://github.com/user-attachments/assets/bca5621a-81cd-4fe8-a7eb-8291bfc91fed"> The Welcome page is squished here with the side bar nav, please fix this.

Reply:
What display size/resolution screen are you using?

Reply:
This should be resolved with the latest push.